cmake/modules/FindSleef.cmake

```
@@ -61,10 +61,28 @@ find_library(Sleef_LIBRARY
   PATHS ${PC_Sleef_LIBRARY_DIRS})
 mark_as_advanced(Sleef_LIBRARY)
 
-find_library(SleefDFT_LIBRARY
-  NAMES sleefdft
-  PATHS ${PC_Sleef_LIBRARY_DIRS})
-mark_as_advanced(SleefDFT_LIBRARY)
+foreach(_component ${Sleef_FIND_COMPONENTS})
+  find_library(Sleef_${_component}_LIBRARY
+    NAMES sleef${_component}
+    PATHS ${PC_Sleef_LIBRARY_DIRS})
+  mark_as_advanced(Sleef_${_component}_LIBRARY)
+  set(ADDITIONAL_REQUIRED_VARS ${ADDITIONAL_REQUIRED_VARS} Sleef_${_component}_LIBRARY)
```

><PackageName>_FIND_COMPONENTS
>    List of specified components (required and optional)

so I think we need to add this: 

```suggestion
  if((Sleef_FIND_REQUIRED_${_component})
    set(ADDITIONAL_REQUIRED_VARS ${ADDITIONAL_REQUIRED_VARS} Sleef_${_component}_LIBRARY)
  endif()
```